mainapp/templates/account/email_confirm.html

```diff
@@ -0,0 +1,39 @@
+{% extends "account/base.html" %}
```

Comment:
Can we just use base.html instead of account/base.html? (If we can, please apply the same change to the following templates)